pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -4179,23 +4174,6 @@ def TruckingCostRule(model, l, l_tilde, t):
         model.s_L, model.s_L, model.s_T, rule=TruckingCostRule, doc="Trucking cost"
     )
 
-    def TotalTruckingFreshFlowRule(model):
```

Comment:
Note: this constraint was not in documentation originally. Removed because it set piping from Freshwater to Disposal to 0 which is no longer needed with new initializatino for V_F_Piped and v_F_Trucking. No update to documentation needed/